src/components/authenticationPage.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react';
+
+class WaitingPage extends Component {
+    
+async postData(url = '', data = {},token) {
+
+    const response = await fetch(url, {
+      method: 'POST', 
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorisation':token
+      },
+      redirect: 'follow', // manual, *follow, error
+      referrerPolicy: 'no-referrer', // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
+      body: JSON.stringify(data) // body data type must match "Content-Type" header
+      });
+    return response.json(); // parses JSON response into native JavaScript objects
```

Comment:
Does the orchestration service return a json response?

Reply:
this is to see the respond of if its succeeded in console or not so atm its printing out "invalid Authorisation token" as the orchestration service isn't validating it correctly.